packages/plugin-hardhat/src/utils/options.ts

```diff
@@ -2,11 +2,15 @@ import { ValidationOptions, withValidationDefaults } from '@openzeppelin/upgrade
 
 export interface Options extends ValidationOptions {
   constructorArgs?: unknown[];
+  timeout?: number;
+  pollingInterval?: number;
```

Comment:
We can't extend two interfaces simultaneously but we can use an intersection type to define `Options`:

```suggestion
export type Options = ValidationOptions & DeployOpts & {
  constructorArgs?: unknown[];
```

Note: The docstrings in this case should be in the definition of DeployOpts.